package.json

```diff
@@ -52,7 +52,7 @@
     "typescript": "^3.9.5"
   },
   "lint-staged": {
-    "*.src.*{ts,js}": [
+    ".*src.*{ts,js}": [
```

Comment:
What does this do?

Reply:
there was a bug (dot was in a wrong place).